### PR TITLE
Move DB, use [[ instead of test

### PR DESCRIPTION
**Pull request by kyrias**

### CODE REVIEW

**Comment by x0054** [comment]
I am not a fan of single line if statement, but it is the in thing to do, so why not. I do agree that pouting the home directory isn't nice, but every one else does it :) I'll merge the commit, but if we are going to move the db location, might as well move it for good. As for the [ vs [[, I don' think I am using any features that [[ offers, but I think it might be a little faster?

File: hyperjump [hunk 1]
```diff
@@ -149,19 +148,16 @@ function jj() {
 		local param
 		for param in ${@:2}
 		do
-			if [ ! -z "$param" -a "$param" != "" ]
-			then
+			if [[ ! -z "$param" -a "$param" != "" ]]; then
```
Comment (x0054) [inline hyperjump:151]: If you are going to change to [[, you also need to change -a to &&. Did you try running the changes?

Comment (kyrias) [inline hyperjump:151]: Ah, only tried it in zsh, will fix that.

**Comment by kyrias** [comment]
> I am not a fan of single line if statement, but it is the in thing to do, so why not. 

I can change back to two-line ifs if you want though, but is there any reason you prefer them?

> I do agree that pouting the home directory isn't nice, but every one else does it :)

No software I use need things in `$HOME` anymore. Most support the XDG basedir spec natively, and zsh can easily have the configs moved properly.

> As for the [ vs [[, I don' think I am using any features that [[ offers, but I think it might be a little faster?

True, the only "advantage" to hyporjump, other than `[` being deprecated, was that you used unquoted variables, and `[[` doesn't do word splitting, but there is really no reason to ever use `[` anymore, all it does is help people write broken scripts.

---

And yeah, it should have been `$old_db` there.

File: hyperjump (hunk 1 above)
Comment (x0054) [inline hyperjump:151]: It works in zsh, hmm, I should try zsh one of these days. I am still running the good old bash :)

**Comment by x0054** [comment]
My main experience with writing shell scripts was actually writing them for sh and sh does not support [[. Hence also the use of ` in a lot of places. It's been a while since I wrote anything in bash. So all the changes you made are actually pretty good. Thanks!

As for the 2 line if statement, don't worry about it, it looks cleaner that way. It's just an old habit of mine :)

**Pull request merged**